### PR TITLE
deprecate lena

### PR DESCRIPTION
**Pull request by johnnychen94**
The first step to deprecate the usage of lena #72

### CODE REVIEW

File: src/TestImages.jl [hunk 1]
```diff
@@ -77,6 +77,10 @@ $(reduce((x, y)->join([x, "\n - \`\"", splitext(y)[1], "\"\`"]), sort(remotefile
 function testimage(filename; download_only::Bool = false, ops...)
     imagefile = image_path(full_imagename(filename))
 
+    if startswith(basename(imagefile), "lena_")
+        @warn "For copyright issue, it is not recommended to use \"lena\" image. Please use other images instead." maxlog=1
```
Comment (timholy) [inline src/TestImages.jl:81]:
```suggestion
        @warn "Usage of \"lena\" is not recommended, and the image may be removed in a later release. See https://womenlovetech.com/losing-lena-why-we-need-to-remove-one-image-and-end-techs-original-sin/ for more information." maxlog=1
```

**Comment by johnnychen94** [comment]
Nightly test on macOS actually now passes locally with `Version 1.6.0-DEV.764 (2020-08-29)`

I'll wait for a few more days to see if things go well in CI.

File: src/TestImages.jl [hunk 2]
```diff
@@ -77,6 +77,10 @@ $(reduce((x, y)->join([x, "\n - \`\"", splitext(y)[1], "\"\`"]), sort(remotefile
 function testimage(filename; download_only::Bool = false, ops...)
     imagefile = image_path(full_imagename(filename))
 
+    if startswith(basename(imagefile), "lena_")
+        @warn "Usage of \"lena\" is not recommended, and the image may be removed in a later release. See https://womenlovetech.com/losing-lena-why-we-need-to-remove-one-image-and-end-techs-original-sin/ for more information." maxlog=1
```
Comment (johnnychen94) [inline src/TestImages.jl:81]: Here we use an explicit warning instead of a depwarn because depwarn is by default not showing up after Julia 1.5.

**Review by johnnychen94 — COMMENTED** [review]
There are a lot of papers using the Lena image, reproducing experiment results is still a practical reason to not remove this. My preference for this is to make this a permanent warning.

Whether remove it or not, we need to deprecate it first.

**Pull request merged**